src/pages/SignUp/SignUp.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useEffect } from 'react';
+import { Navigate, useNavigate } from 'react-router-dom';
+import './SignUp.scss';
+
+const SignUp = () => {
+  const navigate = useNavigate();
+
+  const [formValue, setFormValue] = useState({
+    email: '',
+    password: '',
+    checkPassword: '',
+    userName: '',
+    phoneNumberCenter: '',
+    phoneNumberLast: '',
+  });
+
+  const handleForm = e => {
+    const { name, value } = e.target;
+    setFormValue({
+      ...formValue,
+      [name]: value,
+    });
```

Comment:
크으으... 👍🏻👍🏻👍🏻 계산된 속성명에 치인다...!
prettier가 화 안낸다면 한줄로 쓰는게 가독성에 더 좋아보입니다!! 

```suggestion
    setFormValue({ ...formValue, [name]: value });
```